adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -108,6 +109,7 @@ public void export(final CannotConstructDataProviderNotice toExport) throws IOEx
         protoBuilder.setCsvFileName(toExport.getFilename())
                 .setType(GtfsValidationOutputProto.GtfsProblem.Type.TYPE_CSV_UNKNOWN_ERROR)
                 .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.ERROR)
+                .clearAltEntityId()
```

Comment:
Good catch! As we said together, replace this by a call to `clear()` at the beginning of the chain to clear the reused internal builder